docs/mastering-plone-5/instructions_plone5/index.md

```diff
@@ -0,0 +1,382 @@
+---
+myst:
+  html_meta:
+    "description": "Set up environment for development"
+    "property=og:description": "Set up environment for development"
+    "property=og:title": "Installing Plone for the Training"
+    "keywords": "Plone 5"
+---
+
+(plone5-instructions-label)=
+
+# Installing Plone for the Training
+
+Keep in mind that you need a fast Internet connection during installation since you'll have to download a lot of data!
+
+(plone5-instructions-no-vagrant-label)=
```

Comment:
This should be moved downward to be just above the heading.

Reply:
done. thanks for the hint!